setup.sh

```diff
@@ -55,7 +55,8 @@ apt -y install build-essential autoconf autotools-dev
 
 apt -y install emacs \
 	geany gedit joe kate kdevelop nano vim vim-gtk3 \
-	ddd valgrind visualvm ruby python3-pip konsole
+	ddd valgrind visualvm ruby python3-pip konsole \
+	libqt5gui5-gles libqt5quick5-gles clangd
```

Comment:
It is mentioned here: https://github.com/ioi-2023/contestant-vm/issues/9#issuecomment-1640214092

Reply:
Basically the issue is unintentional removal of some select packages due to changes in dependencies. I'll try to do a mark-as-unneeded instead of remove, and then use autoremove to prevent accodentally uninstalling other packages. This is described in the issue too

Thanks for the investigation